src/clj_icat_direct/icat.clj

```diff
@@ -247,6 +247,13 @@
    (let [group-ids-query (str "SELECT group_user_id FROM (" (q/mk-groups user zone) ") g")]
      (get-item* dirname basename group-ids-query))))
 
+(defn- build-groups-table-query [user-group-ids]
+  (when user-group-ids
+    (apply str
+           "SELECT column1 AS group_user_id FROM (VALUES "
+           (string/join ", " (map #(str "(" % ")") user-group-ids))
+           ") v")))
```

Comment:
The primary purpose for adding this function was for formatting. There should be no functional change.